### PR TITLE
fix module cache breaking for modules not being mocked

### PR DESCRIPTION
**Pull request by connorjclark**
This resolves https://github.com/testdouble/testdouble.js/issues/492 without breaking any existing tests.

Confirmed this work great in our usage of testdouble in Lighthouse tests

### CODE REVIEW

File: lib/quibble.mjs [hunk 1]
```diff
@@ -80,8 +99,8 @@ function getStubsInfo (moduleUrl) {
   if (!moduleUrl.searchParams.get('__quibble')) return undefined
 
   const moduleFilepath = moduleUrl.pathname
-
-  return [...global.__quibble.quibbledModules.entries()].find(([m]) => m === moduleFilepath)
+  const stub = global.__quibble.quibbledModules.get(moduleFilepath)
+  return stub ? [moduleFilepath, stub] : undefined
```
Comment (connorjclark) [inline lib/quibble.mjs:103]: drive-by performance improvement

Comment (searls) [reply lib/quibble.mjs:103]: always appreciated. this lib doesn't get a lot of love

**Comment by searls** [comment]
Hand-merged this one after confirming locally b/c CI was disconnected. Thanks!

Landed in 0.6.10

**Comment by searls** [comment]
This was reverted in 0.6.11. See discussion in https://github.com/testdouble/testdouble.js/issues/493
